vehicle/mkvi/software/imu/imu.yml

```diff
@@ -0,0 +1,77 @@
+name: imu
+publish:
+  - name: imu_accel
```

Comment:
just doing some quick math, 48 bits * 326 messages / sec * 2 messages roughly equals 31 kbps. probably isn't a concern based on our current can bus (I think you said we we're at 25% utilization at some point?), but is something we should probably keep in mind if we have an array of IMUs / add other high frequency messages

also have you tested that your can messages are actually being sent at that frequency? id be impressed if the atmega can actually handle that since the can functions (and I think SPI?) functions are blocking

Reply:
I agree, and I think given our current hardware it'd be wise to keep it all on a separate bus, which is what the harness I sent acts as.

I didn't measure anything rigorously, I only adjusted the main timer compare match such that the reported CAN bus load from `canbusload` was around 9% per IMU. I sent the tag connect with the IMUs so this can be updated if need be. The same applies if we get garbage data when reporting data too fast.

In terms of sending data in the 200-400Hz range, I believe the Atmega and it's CAN peripheral can handle it. Reporting at 300Hz gives the Atmega 3.3ms to perform 12 8-bit SPI reads @ 4MHz and send 3 CAN messages @500KHz. So I'd estimate around  (3.3ms - (12 * 8)/4M - (64 * 3)/500k) = 2.9ms of overhead. That of course with a lot of assumptions - no overhead for CAN messages or SPI, etc.